frontend/src/lib/components/ActionSelectTab.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { capitalizeFirstLetter } from '~/lib/utils'
+
+export default class ActionSelectTab extends Component {
+    render() {
+        let { entityType, chooseEntityType, allTypes } = this.props
+        return (
+            <div style={{ display: 'flex', flexDirection: 'row', height: '25px', borderBottom: '1px solid #cccccc' }}>
+                {allTypes.map((type, index) => (
+                    <div
+                        key={index}
+                        style={{
+                            backgroundColor: entityType == type ? 'white' : '#eeeeee',
```

Comment:
I think this should be the other way around, it really confused me ha. Would also be good to add `cursor: pointer`.